### DISCUSSION TITLE
Welcome to elixirschool Discussions!

### DISCUSSION THREAD

**Discussion by doomspork** [doomspork/elixir-school#2551] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jaimeiniesta** [comment]
Great idea! My name is Jaime and I'm an independent product builder focused now in Rocket Validator. I contributed writing the lesson about NimblePublisher and its translation to Spanish, I'd love to help with the migration of elixirschool to Elixir 😺

**Reply by doomspork** [reply]
Hooray! I'd love some help. I'm currently doing that work in another repo so as not to clobber people's hard work here: https://github.com/elixirschool/school_house

**Comment by atmorojo** [comment]
Hello, my name is Risyad and I'm a freelance programmer. I'm new to Elixir so obviously I'm starting from basics and I'm trying to help with indonesian translation while I'm at it.

This is my second time working on a translation project and I think having a glossary helps a lot. Is it possible to set a GLOSSARY.md inside a language directory to make navigating through it easier (i.e not mixed with another language)?

**Reply by doomspork** [reply]
@atmorojo a glossary is a really great idea!
